### PR TITLE
Add support for DEFAULT_ORG_MEMBER_VISIBLE

### PR DESCRIPTION
**Pull request by guillep2k**
This PR is intended to implement the second behavior suggested on #7512.

DEFAULT_ORG_MEMBER_VISIBLE = true will make the members visible when added to organisations. It's pretty self-explanatory.

### CODE REVIEW

**Comment by guillep2k** [comment]
Sorry, I don't really understand the nature of the failure at [docker-linux-amd64 — dryrun
](https://drone.gitea.io/go-gitea/gitea/11270/3/2).

```
[...]
go: finding github.com/Unknwon/i18n v0.0.0-20171114194641-b64d33658966
go: error loading module requirements 
modules/options/options.go:8: running "go": exit status 1 
[...]
```

Anyone can point me in the right direction?

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 1]
```diff
@@ -243,6 +243,7 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `SHOW_REGISTRATION_BUTTON`: **! DISABLE\_REGISTRATION**: Show Registration Button
 - `AUTO_WATCH_NEW_REPOS`: **true**: Enable this to let all organisation users watch new repos when they are created
 - `DEFAULT_ORG_VISIBILITY`: **public**: Set default visibility mode for organisations, either "public", "limited" or "private".
+- `DEFAULT_ORG_MEMBER_VISIBLE`: **false** Set default visibility for organisation members when added.
```
Comment (lunny) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]: True will make the membership of the users visible when added to the organisation

Comment (guillep2k) [reply docs/content/doc/advanced/config-cheat-sheet.en-us.md]: Done!

**Review by sapk — COMMENTED** [review]
Can you add a case on the already existing test ? 
You can add a params for expected visibility to the func
https://github.com/go-gitea/gitea/blob/f9ec2f89f2265bc1371a6c62359de9816534fa6b/models/org_test.go#L432
and switch the visibility before a new test execution:
https://github.com/go-gitea/gitea/blob/f9ec2f89f2265bc1371a6c62359de9816534fa6b/models/org_test.go#L445

**Comment by guillep2k** [comment]
@sapk 👍

**Review by sapk — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
Make LG-TM work :)

**Pull request merged**